server/static/files.go

```diff
@@ -0,0 +1,8 @@
+// File built without static files
+package static
+
+import "net/http"
+
+func ServeHTTP(http.ResponseWriter, *http.Request) {}
+
+func Hash(string) string { return "" }
```

Comment:
Is this file used?

Reply:
Yeah it is, the Makefile does this at runtime by copying files.stub.go, but since I don't rely on the Makefile this needs to be explicit. 

This doesn't change any of the existing behaviour though, just means the Makefile doesn't have to do the runtime copy.